hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeVertex.java

```diff
@@ -462,10 +464,14 @@ public <V> VertexProperty<V> property(
             E.checkArgument(!this.hasProperty(propertyKey.id()),
                             "Can't update primary key: '%s'", key);
         }
+        if (value == null) {
+            this.removeProperty(propertyKey.id());
+            return EmptyVertexProperty.instance();
+        }
```

Comment:
prefer to move this block into PR https://github.com/apache/incubator-hugegraph/pull/206 for consistent logic.

Reply:
> prefer to move this block into PR #206 for consistent logic.

Not quite clear, something wrong?

<img width="904" alt="image" src="https://user-images.githubusercontent.com/17706099/212523208-4da913fa-75de-4b74-aa8c-c354ee7eb627.png">

Reply:
that's ok if it has been cherry-picked